calendar/endpoints.bal

```diff
@@ -17,14 +17,22 @@
 import ballerina/http;
 import ballerina/jwt;
 
-# Client for Google Calendar connector.
+# Ballerina Google Calendar connector provides the capability to access Google Calendar API.
+# The connector let you perform calendar and event management operations.
 # 
 # + calendarClient - HTTP client endpoint
 @display {label: "Google Calendar", iconPath: "logo.png"}
-public client class Client {
-    public http:Client calendarClient;
-    private ClientOAuth2ExtensionGrantHandler clientHandler;
+public isolated client class Client {
+    private final http:Client calendarClient;
+    private final ClientOAuth2ExtensionGrantHandler clientHandler;
 
+    # Initializes the connector. During initialization you can pass either http:BearerTokenConfig if you have a bearer
```

Comment:
To initialize, you can pass either the http:BearerTokenConfig if you have a bearer